templates/firefox-ios.xliff

```diff
@@ -3932,6 +3932,14 @@ Private Tab</source>
         <source>Default Search Engine</source>
         <note>Accessibility label for default search engine setting.</note>
       </trans-unit>
+      <trans-unit id="Settings.Search.Accessibility.LearnAboutSuggestions.v124" xml:space="preserve">
+        <source>Learn more about %@ Suggest</source>
+        <note>Accessibility label for Learn more about Firefox Suggest. Placeholder is for the app name - Firefox.</note>
+      </trans-unit>
+      <trans-unit id="Settings.Search.DefaultSearchEngine.Footer.v122" xml:space="preserve">
```

Comment:
Is this really a v122 string? because we're already way into beta for this string

Reply:
Yes, we're aware that the translation may not be ready but this string is related to the work for the feature that is targeted for v122. thanks for being thorough!

Reply:
Yes, this was a last minute addition and I clarified that, due to it's late entry, it's likely to not get a translation quickly. Design & PM said this was fine.

Reply:
I see. Will this string be uplifted to v122? or will it just ride the trains and land with v123?

Reply:
@cyndichin @adudenamedruby: flagging as I think my last comment fell through the cracks!

Reply:
anything for Firefox suggest doesn't need to be uplifted as it shouldn't be available till 124. :)